### ISSUE TITLE
Implement file service size

### ISSUE THREAD

**Issue by TonyRice** [storyscript/runtime#496]
It would be very useful to create a way to determine the size of a file on the filesystem

**Comment by wilzbach** [comment]
Maybe this can be generalized with other infos like last accessed and created?
Also, might be interesting for folders as well.

**Comment by TonyRice** [comment]
yeah I think that would be a lot more useful, general file/folder info. I'll add that to the list

**Comment by judepereira** [comment]
@TonyRice will you be doing this this sprint? If yes, please add it to the current sprint. If not, please unassign yourself (so that somebody else can take over when free).
